.drone.yml

```diff
@@ -33,6 +33,8 @@ steps:
       - pip install -r requirements.txt
       - pip install -e .[dev,doc]
       - cd "$DRONE_WORKSPACE/opencti-platform/opencti-graphql"
+      - npm install -g corepack
+      - cp $DRONE_WORKSPACE/.yarnrc.yml .
```

Comment:
Why do we need to cp the .yarnrc.yml, does it means that locally we will need to cp this file before yarn install too ?

Reply:
ah ok I see, yes we had some concurrency issues on build.... migh be better after github workflow move.